test/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -80,6 +80,27 @@ public static UnitType carrier(final GameData data) {
     return unitType(Constants.UNIT_TYPE_CARRIER, data);
   }
 
+  // Some units hard coded here rather than placed in Constants at the request of community members Ref: Pull Request 1074
```

Comment:
Appreciate the change. I still think it is distracting. This is a good item for us to discuss here though. Why are you of the opinion that these really must be defined in a Constants class. As is, we have some benefits:
- the values are not shared with anything else, so constant does not help remove duplication
- we have better cohesion with the test data being in the test source
- moving to constants would create a layer of indirection, not sure how that is useful since the test data is static.
- These constants are used in test only code for now. If it were in Constants.java, then the assumption would be that we use the values in production code.